utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/JsonSchemaBuilder.scala

```diff
@@ -6,11 +6,11 @@ import org.json.JSONObject
 
 object JsonSchemaBuilder {
 
-  def parseSchema(rawJsonSchema: String): Schema =
+  def parseSchema[T <: Schema](rawJsonSchema: String): T =
```

Comment:
I can't see any invocation different than `parseSchema[Schema]` so why do we need this generic here?

Reply:
yeah, maybe it'd be better to have two methods, if we use `parseSchema[Schema]` in > 90% cases and it's just casting anyway ;)

Reply:
I've changed it a little bit.